SCClassLibrary/JITLib/Patterns/Pdef.sc

```diff
@@ -364,9 +366,13 @@ TaskProxy : PatternProxy {
 			}
 		}
 	}
+
 	wakeUp {
-		if(this.isPlaying) { this.play(quant:playQuant) }	}
+		if(this.isPlaying) { this.play(quant:playQuant) }
+	}
+
 	asProtected {
+		// will be deprecated
```

Comment:
??

Reply:
I'd like to remove this in the future, but not post a warning yet.

Reply:
why?

Reply:
because the method isn't really needed anymore.

Reply:
is this OK?

Reply:
ping …